backend/mora/service/detail_reading.py

```diff
@@ -612,14 +612,18 @@ def get_detail(type, id, function):
         funktionsnavn=handlers.FUNCTION_KEYS[function],
     )
 
-    # TODO: the logic encoded in the functions below belong in the
-    # 'mapping' module, as part of e.g. FieldTuples
     def is_primary(effect):
         return [
             ext.get('primær', False)
             for ext in mapping.ORG_FUNK_UDVIDELSER_FIELD(effect)
         ]
 
+    def is_fraction(effect):
```

Comment:
Fraction is not a Boolean, so this function doesn't return one; hence, `get_fraction` is more appropriate.